src/test/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println(getName() + " is working.");
```

Comment:
The message should indicate what type of `Machine` is working right now. Instead of just using `getName()`, it should specify that a `Bulldozer` is working.